docs/Developers/Newcomers_Guide.rst

```diff
@@ -124,45 +127,45 @@ Step 3. Picking Up an Issue
 
 Now it is time to pick an issue.
 It is the best way to familiarise yourself with the codebase.
-Here is the link that will lead you to `Newcomers issues <https://coala.io/new>`_.
+You can view `all Newcomer issues on GitHub <https://coala.io/new>`_.
 
 .. note::
 
-    You need to be logged in before you follow the Newcomers issues link.
+    You need to be logged in before you follow the Newcomer issues link.
 
 .. seealso::
 
     For more information about what bears are, please check the following link: `Writing Native bears <http://api.coala.io/en/latest/Developers/Writing_Native_Bears.html>`_
 
 The easy issues that will help you get started are labeled as
-``difficulty/newcomer`` and are only there to give you a glimpse of how it is
-to work with us and regarding the workflow.
+``difficulty/newcomer`` and are only there to give you a glimpse of what
+it's like to work with us and what the workflow is like.
 
-Now pick an issue which isn't assigned, and if you want to fix
-it, then leave a comment that you would like to get assigned. This way
+Now pick an issue which isn't assigned and which you would like to fix.
+Leave a comment that you would like to be assigned to this issue. This way
```

Comment:
assigned to is the correct way :), but I would say `the issue` instead of `this issue`